components.yaml

```diff
@@ -1,22 +1,29 @@
 components:
-  multus:
+  - name: "multus"
     url: "https://github.com/intel/multus-cni"
-    commit: "bfaf22964b51b93b08dcb1f10cbd8f8cb9a3603a" # v3.4.1
-  linux-bridge:
+    commit: "bfaf22964b51b93b08dcb1f10cbd8f8cb9a3603a"
+    release_tag: "v3.4.1"
+  - name: "linux-bridge"
     url: "https://github.com/containernetworking/plugins"
-    commit: "ad10b6fa91aacd720f1f9ab94341a97a82a24965" # v0.8.6
-  bridge-marker:
+    commit: "ad10b6fa91aacd720f1f9ab94341a97a82a24965"
+    release_tag: "v0.8.6"
+  - name: "bridge-marker"
     url: "https://github.com/kubevirt/bridge-marker"
-    commit: "049eb6a796b742767a0be838f6ddb7f79c97dd60" # 0.3.0
-  kubemacpool:
+    commit: "049eb6a796b742767a0be838f6ddb7f79c97dd60"
+    release_tag: "0.3.0"
+  - name: "kubemacpool"
     url: "https://github.com/k8snetworkplumbingwg/kubemacpool"
-    commit: "7728339b6c8dcfc9d53cc18ce8e06a910094ed47" # v0.16.0
```

Comment:
Do we really need this ? it add extra changes here and on the bash scripts

Reply:
at the very least we need to change components names so thary they don't have `-` (cannot parse the yaml otherwise)

Reply:
Can we keep the name in quotes to prevent that issue? It is not as elegant but would allow us to move forward without breaking existing code.

Reply:
do you mean like this?
```
  "macvtap-cni":
    url: "https://github.com/kubevirt/macvtap-cni"
    commit: "14e5a3c7bf516bd4542366a78b7af02f04230ac5"
```

Reply:
You can specify in the structure attribute what is the name of it's name of the equivalent in yaml/json. Doesn't that work? And even then, there is still option to treat this as a map.